src/dex-ui/services/HederaService/HederaService.ts

```diff
@@ -40,17 +42,22 @@ function createHederaService() {
     return _precision;
   };
 
+  const queryContract = async (
```

Comment:
`ContractCallQuery` calls were consolidated into a single function to simplify the HederaService code and apply consistent gas payment amounts.